src/components/Trade/TradeConfirm/AssetToAsset.tsx

```diff
@@ -1,38 +1,31 @@
 import { ArrowForwardIcon, CheckIcon, CloseIcon } from '@chakra-ui/icons'
-import {
-  AvatarProps,
-  Box,
-  Circle,
-  Divider,
-  Flex,
-  FlexProps,
-  Text,
-  useColorModeValue,
-} from '@chakra-ui/react'
-import { chainAdapters } from '@shapeshiftoss/types'
+import { Box, Circle, Divider, Flex, FlexProps, Text, useColorModeValue } from '@chakra-ui/react'
+import { Trade } from '@shapeshiftoss/swapper'
+import { chainAdapters, ChainTypes } from '@shapeshiftoss/types'
 import { AssetIcon } from 'components/AssetIcon'
 import { useLocaleFormatter } from 'hooks/useLocaleFormatter/useLocaleFormatter'
-import { bn } from 'lib/bignumber/bignumber'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import { fromBaseUnit } from 'lib/math'
 
-import { TradeAsset } from '../Trade'
-
-type AssetToAssetProps = {
-  sellAsset: TradeAsset
-  buyAsset: TradeAsset & Pick<AvatarProps, 'boxSize'>
+type AssetToAssetProps<C extends ChainTypes> = {
+  tradeFiatAmount: string
+  buyIcon: string
   status?: chainAdapters.TxStatus
+  trade?: Trade<C>
 } & FlexProps
 
 export const AssetToAsset = ({
-  sellAsset,
-  buyAsset,
+  tradeFiatAmount,
+  buyIcon,
+  trade,
   boxSize = '24px',
   status,
   ...rest
-}: AssetToAssetProps) => {
+}: AssetToAssetProps<ChainTypes>) => {
```

Comment:
If we're not using a specific `ChainTypes`, might as well default the generic

Reply:
Not sure how to do that

IMO what I have here works fine but willing to change it if theres a better way

Reply:
```typescript
type AssetToAssetProps<C extends ChainTypes = ChainTypes> = {
``` 

Tried this locally and it works, no need to pass a generic